artifactory.py

```diff
@@ -2585,15 +2585,16 @@ def get_build_info(self, build_name, build_number):
         return self._get_info(build_name, build_number)
 
     def _get_info(self, build_name, build_number=""):
-        url = build_name
+        url = requests.utils.quote(build_name, safe="")
         if build_number:
             url += f"/{build_number}"
         return self._get_build_api_response(url)
 
     def _get_build_api_response(self, url):
         url = f"{self.drive}/api/build/{url}"
-        obj = self.joinpath(url)
-        resp = self._accessor.get_response(obj).json()
+        response = self.session.get(url)
```

Comment:
Why do we need to change that part? I don't remember what it's doing, but it should affect all code, I guess, so just checking  it out

Reply:
if we use `self._accessor.get_response()`, the url will be encoded again.
For example,  `maven-demo%2F1-build-snapshot` would be encoded to `maven-demo%252F1-build-snapshot`.
So I changed the part.

Reply:
As I see `_accessor` does some additional work - passing certificates and other configuration. 
We shouldn't just remove it from here, let's communicate with API by accessor 

https://github.com/devopshq/artifactory/blob/1a3490867344165ca4c2fbcc7ce0e5b13e096368/artifactory.py#L1082-L1101

Reply:
ok, I added the param `skip_quote_url` for `get_response()` and `rest_get_stream()` methods.